ducktape/services/service_registry.py

```diff
@@ -52,7 +52,7 @@ def stop_all(self):
             except BaseException as e:
                 if isinstance(e, KeyboardInterrupt):
                     keyboard_interrupt = e
-                service.logger.warn("Error stopping service %s: %s" % (service, e.message))
+                service.logger.warn("Error stopping service %s: %s" % (service, e))
```

Comment:
nit: you could use ',' for variadic calls rather than interpolating directly.

Reply:
Was just doing minimal conversion (I think there are a bunch of other cases of this as well), but have updated this one. There's probably a bunch of other cleanup we could do as well, especially as some style things changed from py2 -> 3.